R/http.R

```diff
@@ -336,7 +306,9 @@ urlDecode <- function(x) {
 }
 
 urlEncode <- function(x) {
-  if (inherits(x, "AsIs")) return(x)
+  if (inherits(x, "AsIs")) {
+    return(x)
+  }
```

Comment:
I'm confused that this function works, given that it depends on RCurl, which is only in suggests.

Reply:
Also, sometimes we use `utils::urlEncode` and the the private `urlDecode` uses `curl::curl_unescape`.

A partial shift away from RCurl happened in https://github.com/rstudio/rsconnect/pull/348, and RCurl was still an import dependency at that point in time, moving to suggests in https://github.com/rstudio/rsconnect/commit/35ba0497f7266da2c9157cbcd1b55e29c2876b4d

Also surprised.